js/data/locale/currency.jf

```diff
@@ -1 +0,0 @@
-{"currency": "USD"}
```

Comment:
The root currency should be US Dollars if no others are specified

Reply:
I'll rollback the file. I'll update the extractor tool code so that it can be automatically generated from the tool code.